packages/suite/src/utils/wallet/coinmarket/exchangeUtils.ts

```diff
@@ -24,9 +25,12 @@ export const getAmountLimits = (quotes: ExchangeTrade[]): AmountLimits | undefin
         if (!quote.error && noError) {
             return;
         }
+        if (!currency && quote.send) {
+            currency = quote.send;
+        }
     }
     if (min || max) {
-        return { currency: quotes[0].send || '', min, max };
+        return { currency: currency || '', min, max };
```

Comment:
```suggestion
        return { currency, min, max };
```
currency already assigned in line 11

Reply:
Done